build.psm1

```diff
@@ -347,7 +347,9 @@ function Start-PSBuild {
         Write-Warning @"
```

Comment:
@iSazonov, your last commit had 7 failures in `PowerShell-CI-linux`
(These are 5 of the failures)

Export-FormatData.Can export all types
```powershell
Expected $null, but got @(@{InputObject=Microsoft.PowerShell.Commands.X509StoreLocation; SideIndicator==>}, @{InputObject=System.Security.Cryptography.X509Certificates.X509Certificate2; SideIndicator==>}, @{InputObject=System.Security.Cryptography.X509Certificates.X509Store; SideIndicator==>}, @{InputObject=System.CodeDom.Compiler.CompilerError; SideIndicator==>}, @{InputObject=System.Reflection.Assembly; SideIndicator==>}, @{InputObject=System.Reflection.AssemblyName; SideIndicator==>}, @{InputObject=System.Globalization.CultureInfo; SideIndicator==>}, @{InputObject=System.Diagnostics.FileVersionInfo; SideIndicator==>}, @{InputObject=System.Diagnostics.EventLogEntry; SideIndicator==>}, @{InputObject=System.Diagnostics.EventLog; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Export-FormatData.Tests.ps1: line 30
30:             Compare-Object $fd $actualAllFormat | Should -Be $null
```
Select-String.String actions.Should use the ca alias for casesensitive
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Select-String.Tests.ps1: line 48
48:             $equal | Should -BeTrue
```
Stream writer tests.Write-Information cmdlet.Write-Information works with defaults
```powershell
Expected 0, but got 2.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1: line 115
115:             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
```
Stream writer tests.Write-Information cmdlet.Write-Information works with -Object
```powershell
Expected 0, but got 2.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1: line 115
115:             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
```
Stream writer tests.Write-Information cmdlet.Write-Information works with -Message
```powershell
Expected 0, but got 2.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Write-Stream.Tests.ps1: line 115
115:             (Compare-Object $result $returnValue -SyncWindow 0).length | Should -Be 0
```